screens/HomeScreen.js

```diff
@@ -0,0 +1,53 @@
+import {React, useState} from "react";
+
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import Container from "../common/Container";
+import EStyleSheet from "react-native-extended-stylesheet";
+
+
+function HomeScreen({ navigation, route }) {
```

Comment:
Instead of passing in parameters to every screen after the log in, we can use the AWS Amplify function Auth.currentUserInfo(). It gets all the attributes you would need for the profile screen.